apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -215,7 +212,7 @@ public string SpannerInstance
         /// Credentials can be retrieved from a file or obtained interactively.
         /// See Google Cloud documentation for more information. May be null.
         /// </param>
-        public SpannerConnectionStringBuilder(string connectionString, ITokenAccess credential = null)
+        public SpannerConnectionStringBuilder(string connectionString, ChannelCredentials credential = null)
```

Comment:
Everywhere that the parameter is called credential we could *consider* calling it credentials instead, just to match the noun. I'm not overly bothered, but I thought I'd mention it as an option. (Likewise GetCredential would be GetCredentials.)

Reply:
seems reasonable.  I'm with it.